e2e/live/end-to-end-test.sh

```diff
@@ -836,6 +850,163 @@ assertPodExists "pod-c" "test-rg-namespace"
 assertPodExists "pod-d" "test-rg-namespace"
 printResult
 
+###########################################################
+#  Test Migrate from Kptfile inventory to ResourceGroup
+###########################################################
```

Comment:
I'm not crazy about adding more of this type of tests. It has proven very difficult to debug for others. How much effort would it be to instead implement this with the other e2e framework?

Reply:
It will require a quite bit more effort, since we'd have to come up with a solution for building the older kpt binaries from our Go test suite and the logic for testing migrations. But also open to implementing that since it will be a cleaner solution. Alternatively, creating the migrate e2e Go tests in a follow-up PR might be better?